lib/reactive_table.js

```diff
@@ -492,6 +498,14 @@ Template.reactiveTable.helpers({
         if (this.showNavigation === 'always') return true;
         if (this.showNavigation === 'never') return false;
         return getPageCount.call(this) > 1;
+    },
+
+    'showRowCount': function() {
+        if(this.showRowCount)
```

Comment:
I don't think you need to check this if it's already inside an if in the template